tests/Repositories/TaskRepositoryTest.php

```diff
@@ -99,6 +103,115 @@ public function it_can_retrieve_task_list_of_given_projects()
         });
     }
 
+    /** @test */
+    public function it_can_add_comment()
+    {
+        $task = factory(Task::class)->create();
+        $getComment = $this->taskRepo->addComment([
+            'comment' => 'random text',
+            'task_id' => $task->id,
+        ]);
+
+        $this->assertNotEmpty($getComment);
+        $this->assertEquals('random text', $getComment->comment);
+        $this->assertEquals($this->defaultUserId, $getComment->createdUser->id);
+    }
+
+    /** @test */
+    public function test_can_return_unique_task_index()
+    {
+        $project = factory(Project::class)->create();
+        factory(Task::class)->create([
+            'project_id'  => $project->id,
+            'task_number' => 3,
+        ]);
+        $getUniqueIndex = $this->taskRepo->getIndex($project->id);
+
+        $this->assertNotEmpty($getUniqueIndex);
+        $this->assertEquals(4, $getUniqueIndex, '+1 index');
+    }
+
+    /** @test */
+    public function test_can_return_index_one_when_task_does_not_have_task_number()
```

Comment:
convert this test to `test_can_retunn_task_number_one_when_no_tasks_on_given_project`, and change test as per test name